src/map/skill.c

```diff
@@ -7347,8 +7347,12 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 
 	case AL_HOLYWATER:
 		if(sd) {
-			if (skill_produce_mix(sd, skill_id, ITEMID_HOLY_WATER, 0, 0, 0, 1, -1))
-				clif_skill_nodamage(src,bl,skill_id,skill_lv,1);
+			if (skill_produce_mix(sd, skill_id, ITEMID_HOLY_WATER, 0, 0, 0, 1, -1)) {
+				struct skill_unit* su;
+				if ((su = map_find_skill_unit_oncell(bl, bl->x, bl->y, NJ_SUITON, NULL, 0)) != NULL)
+					skill_delunit(su);
```

Comment:
@cydh instead of swearing around you could show @Playtester the confirmation you gathered through Openkore or whatever tool you used. If you do not add any proof it is statement against statement.
As you know for sure aegis implementations differ from server to server from time to time, so it might be good to gather all the facts from both sides before swearing around in public.

As for me personally if it really is statement against statement you might as well consider, that @Playtester is an official team member and therefore his statement might be consider more worthy than the statement of an "inofficial" team member who only uses his commit permissions when he feels like without talking to anyone and additionally only insults and offends people from the team or others.

To sum it up, if I were you I would really reconsider who is the one who has a stuck brain and who has the right to tell someone to fuck off. It would also be a pleasure for all of us to see your manner in public getting a little more respectful. I do not care how you talk about us in private or directly to us, but writing comments like these in public while being in your position is absolutely not okay.

Best regards,
Lemon